### DISCUSSION TITLE
Clarity Combobox: make <clr-option> diasable-able

### DISCUSSION THREAD

**Discussion by xiao-lix** [vmware-clarity/core#68] [General]
We would like to have the same feature request as this [issue](https://github.com/vmware/clarity/issues/6461) bring up - to disable individual options in the clarity combobox component. The original issue is closed and the repo is archived. I pinged internal channel for help and was pointed to open an issue here on github. 

The current implementation does not support disabled attribute on the options, only the whole combobox can be disabled. It would be very useful to make a single option disable-able both visually and functionally including disable clicking and keyboard selection. 

For example: 
![image](https://user-images.githubusercontent.com/20292261/168381501-23203b58-7d32-4369-bfcf-b1bfa89a0dba.png)

**Comment by ashleyryan** [comment]
Hi there, this repo is for the web component implementation of Clarity, prefixed with `cds` - anything with `clr` is the angular library.

Can you repost this in https://github.com/vmware-clarity/ng-clarity?

**Reply by xiao-lix** [reply]
Hi, thanks for the reminder. Yes sure, moved to the other repo thanks!
